compiler/src/iree/compiler/Dialect/LinalgExt/IR/TilingInterfaceImpl.cpp

```diff
@@ -1340,6 +1340,118 @@ LogicalResult TopkOp::getResultTilePosition(
   return success();
 }
 
+//===----------------------------------------------------------------------===//
+// ArgmaxOp
+//===----------------------------------------------------------------------===//
+
+SmallVector<Range> ArgmaxOp::getIterationDomain(OpBuilder &builder) {
+  Location loc = getLoc();
+  OpFoldResult zero = builder.getIndexAttr(0);
+  OpFoldResult one = builder.getIndexAttr(1);
+  SmallVector<Range> ranges;
+  for (int64_t dim = 0; dim < getInputRank(); ++dim) {
```

Comment:
Do not recalculate the trip count: https://llvm.org/docs/CodingStandards.html#don-t-evaluate-end-every-time-through-a-loop